crates/cheatcodes/src/inspector.rs

```diff
@@ -929,7 +924,21 @@ impl Cheatcodes {
                         emitter: log.address,
                     }));
                 }
-                self.combined_logs.extend(result.logs.clone().into_iter().map(Some));
+
+                // append console logs from zkEVM to the current executor's LogTracer
+                let executor = &mut TransparentCheatcodesExecutor;
+                result.logs.iter().filter_map(decode_console_log).for_each(|decoded_log| {
+                    executor.console_log(
```

Comment:
Isn't this a noop?
`TransparentCheatcodesExecutor` returns the `CheatsCtxt.state` (which is `self: Cheatcodes`) below, which uses the default impl for `InspectorExt::console_log` which is `{}` (impl at L1886, default at `crates/evm/core/src/lib.rs`)

Reply:
It should trigger this method in the entire inspector stack where LogTracer picks it up.

Reply:
Oh so it goes thru each inspector in the stack? Ok that's nice